Dockerfile

```diff
@@ -4,7 +4,7 @@ SHELL ["/bin/bash", "-c"]
 # Install Codejail Packages
 RUN apt-get update && apt-get upgrade -y
 RUN apt-get install -y vim python3-virtualenv python3-pip
-RUN apt-get install -y sudo
+RUN apt-get install -y sudo git
```

Comment:
Adding git package for subsequent PR, where we will use it for fetching latest changes to repo while running tests in CI